scripts/zones/Chamber_of_Oracles/npcs/Armoury_Crate.lua

```diff
@@ -67,6 +67,76 @@ local loot =
             {itemid =  1257, droprate =   50}, -- chunk_of_wind_ore
         },
     },
+    -- KSNM Eye of the Storm
+    [198] =
+    {
+        {
+            {itemid = 1124, droprate = 1000}, -- Wyvern Wing
+        },
+        {
+            {itemid = 1122, droprate = 1000}, -- Wyvern Skin
+        },
+          
+        {    
+            {itemid = 65535, droprate = 1000, amount = 4000},  -- 4k Gil, really should be 24k/total number of players. 
```

Comment:
Amount should just be 24k, loot handling will split it automatically.
https://youtu.be/TlvThE0sM14?t=950

Reply:
got it, wasn't sure how that split worked and thought i would need some getplayercount shenanigans i wasnt prepared yet to do. much easier, thank you!